packages/core/src/runtime/Step.ts

```diff
@@ -100,6 +113,23 @@ export function extractOptionsAndCallback(args: any[]): [Partial<StepOptions>, T
  * ```
  */
 export type StepFunction<T> = (driver: Browser, data?: T) => Promise<void>
+export type StepRecoveryObject = {
+	[name: string]: Step
+}
+
+/**
+ * The `RecoveryOption` represents action which we will do after the recovery step has finished
+ * ```
+ * RETRY: retry the step failed
+ * RESTART: re-run the first step
+ * ERROR: throw an error
```

Comment:
Should this be better?
```suggestion
 * Possible actions to do after a step failed
 * ```
 * RETRY: retry the failed step
 * RESTART: re-run the step
 * ERROR: throw an error
```

Reply:
it's good but I think it does not clear as the original one.